lib/mixlib/shellout.rb

```diff
@@ -297,7 +289,6 @@ def parse_options(opts)
           self.password = setting
         when 'user'
           self.user = setting
-          self.with_logon = setting
```

Comment:
No its internal.  The windows code that hooked "self.with_login" got converted to use "self.user".

You can check it out, but it looked like a uselessly duplicated internal variable state.